presto-jdbc/src/main/java/io/prestosql/jdbc/ConnectionProperties.java

```diff
@@ -41,6 +41,16 @@
 
 final class ConnectionProperties
 {
+    enum SslVerificationMode
+    {
+        FULL,
+        CA,
+        NONE;
+
+        static final Predicate<Properties> IF_SSL_VERIFICATION_ENABLED =
+                checkedPredicate(properties -> !SSL_VERIFICATION.getValue(properties).orElse(SslVerificationMode.valueOf(DEFAULTS.get(SSL_VERIFICATION.getKey()))).equals(SslVerificationMode.NONE));
```

Comment:
it would be nice to replace
`SSL_VERIFICATION.getValue(properties).orElse(SslVerificationMode.valueOf(DEFAULTS.get(SSL_VERIFICATION.getKey()))` with new `SSL_VERIFICATION.getValueOrDefault(properties)`, that would require adding new method but it would be much readable.